railties/lib/rails/generators/rails/app/templates/.devcontainer/devcontainer.json.tt

```diff
@@ -0,0 +1,49 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ruby
+{
+  "name": "Ruby",
+  "dockerComposeFile": "docker-compose.yml",
+  "service": "rails-app",
+  "workspaceFolder": "/workspaces/${localWorkspaceFolderBasename}",
+
+  // Features to add to the dev container. More info: https://containers.dev/features.
+  "features": {
+    "ghcr.io/devcontainers/features/github-cli:1": {}
+  },
+
+  "containerEnv": {
+<%- if depends_on_system_test? -%>
+    "CAPYBARA_SERVER_PORT": "45678",
```

Comment:
Doesn't the trailing `,` break if there is only one item?

Reply:
Hm, it doesn't seem to cause any problems for vscode. But I will fix it.